docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Search
+
+The Search module provides a mirrored index of all CMS content that is optimized for search relevance, speed and accuracy. The Search module overrides the default search functionality to query the specialized search index in place of a standard MySQL query. This means all default search operations using the CMS search APIs for posts such as `WP_Query`, `get_posts`, `get_search_form()` and the REST API search endpoints will transparently make use of the search index.
+
+The default Search index and related functionality is provided by the [ElasticPress plugin](https://github.com/10up/ElasticPress).
+
+Content that is indexed in the search index by default:
+
+- Posts
+- Pages
+- Media
+- Custom Post Types (registered with `show_in_search` or `public`)
+- Post's Meta
+- Post's Terms
+- Post's Author
+
+When used in conjunction with the Media Rekognition feature, all images are processed for automatic keyword detection and stored in the search index too.
+
+All documents that are uploaded to the media library are also parsed and indexed. For example, if you upload a PDF file, the PDF content will be read and included in the search index. Searches for keywords and phrases that are included in the document will be then be included in search results.
```

Comment:
I don't think this is enabled by default even if the plugin is installed, see #6

Reply:
Hmm when I tried to activate it locally, it was already activated, so pretty sure it was on by default.

Reply:
Ok cool. Might want to be able to disable as it's a resource hog - especially on local, it can cause chassis to stack up unless you feed it lots of rams.

Reply:
Ok, added a module setting for that!